app/javascript/util/listingUtil.ts

```diff
@@ -72,3 +73,53 @@ export const getListingAddressString = (listing: RailsListing): string => {
     ""
   )
 }
+
+/**
+ * Check if a listing is accepting paper applications
+ * @param {RailsRentalListing | RailsRentalListing} listing
+ * @returns {boolean} returns true if the listing is accepting paper applications, false otherwise
+ */
+export const acceptingPaperApplications = (listing: RailsListing): boolean => {
+  return listing.Accepting_applications_at_leasing_agent || listing.Accepting_applications_by_PO_Box
+}
+
+export const mohcdPaperAppURLBase = "https://sfmohcd.org/sites/default/files/Documents/MOH/"
```

Comment:
do these 3 vars need to be exported? perhaps move them into the `paperApplicationURLs` function?